### ISSUE TITLE
Subscriptions: allow blog owners to specify default delivery options in subscription form

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#785]
It would be nice to add a new parameter to the `extra_data` sent to WordPress.com via `jetpack.subscribeToSite` when a reader subscribes:
https://github.com/Automattic/jetpack/blob/3.0.2/modules/subscriptions.php#L318

If that parameter could be set via a filter and allowed blog owners to set a default delivery schedule (instant, daily, weekly), it would be really useful on blogs with a lot of posts.

Suggested in 1880051-t

**Comment by jeherve** [comment]
Also suggested here:
https://wordpress.org/support/topic/some-more-suggestions-for-jetpack-subscriptions?replies=3&view=all

**Comment by verkoren** [comment]
Also looking for this solution... Any progress?

**Comment by jeherve** [comment]
Also suggested here:
https://wordpress.org/support/topic/can-subscriptions-plugin-be-customized-to-send-all-posts-once-a-day?replies=2&view=all

**Comment by rik113** [comment]
I too am very interested in this addition.

**Comment by jeherve** [comment]
Also suggested here:
https://wordpress.org/support/topic/manage-subscription-news-daily-weekly?replies=1&view=all

**Comment by jeherve** [comment]
Also suggested here:
https://wordpress.org/support/topic/about-sending-emails-to-subscribers?replies=1&view=all

**Comment by jeherve** [comment]
Also suggested here:
https://wordpress.org/support/topic/one-email-digest-with-posts?replies=1&view=all

**Comment by jeherve** [comment]
Also suggested here:
https://wordpress.org/support/topic/how-to-stop-auto-sending-emails-by-jetpack?replies=1&view=all

**Comment by jenhooks** [comment]
Here: 2485318-t

**Comment by maapinho** [comment]
Yep.... please add this feature

**Comment by htdat** [comment]
I'm looking forward to this feature as well. 

I think there is an exception for wptavern.com? This site is a hosted site and it's delivering new posts on daily basis.

**Comment by mtw28** [comment]
Daily is the default.

**Comment by htdat** [comment]
@mtw28 : That's not true. By default, for the hosted sites, Jetpack deliveries new posts immediately. I've just tested in one of my sites.

**Comment by jeherve** [comment]
>  think there is an exception for wptavern.com? This site is a hosted site and it's delivering new posts on daily basis.

There are no exceptions made for any site. You can, however, change your delivery settings for each site you follow here:
https://wordpress.com/following/edit

![screen shot 2016-08-15 at 14 18 47](https://cloud.githubusercontent.com/assets/426388/17664276/3c34e732-62f3-11e6-8f34-4fb0a82cf5de.png)

**Comment by jeherve** [comment]
> instant is the default on self-hosted and daily would be a good (often better) option.

@mtw28 You can change your own default delivery frequency settings here:
https://wordpress.com/me/notifications/subscriptions

**Comment by Eirikg89** [comment]
Any progress on this? Would be so great to be able to set a default delivery schedule.

**Comment by csonnek** [comment]
Requested in 3076683-t

**Comment by dcoleonline** [comment]
Requested again in 958110-hc

Also, this affects WordPress.com Business sites who are using Jetpack's connection. In cases where email-only subscribers could previously manage their delivery frequency when a plugin/theme hadn't been activated, they now cannot do this unless they create a WordPress.com account and manage their subscriptions in the Reader.

**Comment by StefMattana** [comment]
Also suggested here: [https://wordpress.org/support/topic/can-default-email-subscription-be-daily-digest-rather-than-every-post/](https://wordpress.org/support/topic/can-default-email-subscription-be-daily-digest-rather-than-every-post/)

**Comment by htdat** [comment]
here also
https://wordpress.org/support/topic/need-a-jetpack-email-subscription-popup/

**Comment by tbradsha** [comment]
4492560-hc

**Comment by supernovia** [comment]
Requested here too:
https://wordpress.org/support/topic/weekly-or-monthly-emails/

**Comment by nero1** [comment]
This is the main reason why I dont use jetpack subscriptions. Subscribers complain of too many messages and most don't bother to modify the delivery, they just plain unsubscribe (with a hiss). 

I have seen hundreds of posts from wordpress owners where this feature has been requested and it seems to me such a basic thing, yet this thread was opened in 2014 and today (six years later) nothing has been done.  I would pay to have this feature implemented if I could. 

I understand that it takes time and effort to implement this, I just wish it has been done already.

**Comment by tmmbecker** [comment]
2985718-zen

**Comment by supernovia** [comment]
I haven't heard a request for this for a while, but I wouldn't say it's stale, as it would still be a useful feature.

**Comment by jeherve** [comment]
Related discussion: pewaj1-U0-p2

**Comment by jeherve** [comment]
Also in 9854843-zd-a8c

**Comment by jbotte** [comment]
I am a bit perplexed how this hasn't been implemented, if I have a site producing 10 articles a day wouldn't I be risking a large amount of unsubscribes and spam reports?

I counted the steps before a user can set their delivery preferences, if the use isn't currently logged into wordpress.com or has a wordpress.com account it could take up to 7 steps including multiple emails.

forexample -> double optin + checking email again and clicking manage account + creating wordpress.com account + accepting magic link to login to said account -> finally setting preferences

Most users wont even know that they have this option available to them unless they take the time to read the welcome email.

**Comment by jbotte** [comment]
### Attached possible example
sorry for the rushed mockup, just was heading out the door when I originally made it.


<img width="857" alt="Image" src="https://github.com/user-attachments/assets/669e0f98-d8fa-423d-a150-6c0afa80cd4d" />

**Comment by jeherve** [comment]
Thanks for the feedback! 

We've discussed some potential improvements, but we haven't made any changes to the interface / block to reflect that yet. It's definitely something we've considered though. As you mentioned, it becomes useful when you know how much you're going to publish each day / week, and don't want to "spam" your readers.

For now, and until our next iteration on the Subscribe block and overall Subscription flows, you could change the default subscription frequency for all future subscribers on your site by setting the `jetpack_newsletters_publishing_default_frequency` option. For example, you could set posts to be delivered weekly by setting the option to `weekly`:

`wp option update jetpack_newsletters_publishing_default_frequency weekly`

*******

Related discussions / work:

- #35541
- #35672
- pewaj1-U0-p2

**Comment by jbotte** [comment]
This is incredibly helpful, thanks !!
